app/src/main/java/ch/epfl/sweng/SDP/game/drawing/DrawingOnline.java

```diff
@@ -128,20 +144,17 @@ protected void onCreate(Bundle savedInstanceState) {
         timerRef.addValueEventListener(listenerTimer);
         stateRef = Database.getReference(TOP_ROOM_NODE_ID + "." + roomId + ".state");
         stateRef.addValueEventListener(listenerState);
+
+        Database.getReference(TOP_ROOM_NODE_ID + "." + roomId + ".onlineStatus."
```

Comment:
Since it is used here too, it could be wrapped in a separated class (maybe the same wrapping the connectivity code a few lines above)